src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1937,7 +1937,137 @@ await using (var context = InMemoryApplicationDbContext(contextId))
 
-                Assert.True(saved.ReleaseStatuses.First().NotifyReleaseApprovers);
+                Assert.True(saved.ReleaseStatuses.First().NotifyHigherReviewers);
+            }
+        }
+
+        [Fact]
+        public async Task CreateReleaseStatus_HigherReview_SendsEmailIfPublicationOwner()
+        {
+            var release = new Release
+            {
+                Type = ReleaseType.AdHocStatistics,
+                Publication = new Publication { Title = "Test publication" },
+                ReleaseName = "2030",
+                Slug = "2030",
+                Version = 0,
+            };
+
+            var userPublicationOwnerRole = new UserPublicationRole
+            {
+                User = new User { Id = Guid.NewGuid(), Email = "test@test.com" },
+                PublicationId = release.PublicationId,
+                Role = PublicationRole.Owner
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.AddRangeAsync(release, userPublicationOwnerRole);
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var contentService = new Mock<IContentService>(MockBehavior.Strict);
+                var preReleaseUserService = new Mock<IPreReleaseUserService>(MockBehavior.Strict);
+                var userReleaseRoleService = new Mock<IUserReleaseRoleService>(MockBehavior.Strict);
+            
+                contentService.Setup(mock => mock.GetContentBlocks<HtmlBlock>(release.Id))
+                    .ReturnsAsync(new List<HtmlBlock>());
+            
+            
+                userReleaseRoleService.Setup(mock =>
+                        mock.ListUserReleaseRolesByPublication(ReleaseRole.Approver, release.Publication.Id))
+                    .ReturnsAsync(new List<UserReleaseRole>());
+
+                var releaseService = BuildService(
+                    contentDbContext: context,
+                    contentService: contentService.Object,
+                    preReleaseUserService: preReleaseUserService.Object,
+                    userReleaseRoleService: userReleaseRoleService.Object
+                );
+
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id, new ReleaseStatusCreateViewModel
+                        {
+                            PublishMethod = PublishMethod.Scheduled,
+                            PublishScheduled = "2051-06-30",
+                            ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
+                            LatestInternalReleaseNote = "Test internal note",
+                            NextReleaseDate = new PartialDate { Month = "12", Year = "2077" },
+                        });
```

Comment:
I suspect this method isn't working as expected, and that it doesn't actually send the email, or that something's wrong with this unit test.

You haven't set up an `EmailTemplateService` with the expected behaviour (i.e. the method call for the email to be sent).

Without setting it up, it's defaulting to a mock service that has strict behaviour. If the strict mock is called to send the email then it would fail.

The fact that it's not failing, indicates the email isn't getting sent.